scripts/covid19indiaORG/Config.py

```diff
@@ -0,0 +1,177 @@
+ },
+ {
+   "Abbreviation": "ML",
+   "State": "Meghalaya",
+   "API": "https://api.covid19india.org/v4/min/timeseries-ML.min.json"
+ },
+ {
+   "Abbreviation": "MN",
+   "State": "Manipur",
+   "API": "https://api.covid19india.org/v4/min/timeseries-MN.min.json"
+ },
+ {
+   "Abbreviation": "MH",
+   "State": "Maharashtra",
+   "API": "https://api.covid19india.org/v4/min/timeseries-MH.min.json"
+ },
+ {
+   "Abbreviation": "MP",
+   "State": "Madhya Pradesh",
+   "API": "https://api.covid19india.org/v4/min/timeseries-MP.min.json"
+ },
+ {
+   "Abbreviation": "LA",
+   "State": "Ladakh",
+   "API": "https://api.covid19india.org/v4/min/timeseries-LA.min.json"
+ },
+ {
+   "Abbreviation": "KL",
+   "State": "Kerala",
+   "API": "https://api.covid19india.org/v4/min/timeseries-KL.min.json"
+ },
+ {
+   "Abbreviation": "KA",
+   "State": "Karnataka",
+   "API": "https://api.covid19india.org/v4/min/timeseries-KA.min.json"
+ },
+ {
+   "Abbreviation": "JH",
+   "State": "Jharkhand",
+   "API": "https://api.covid19india.org/v4/min/timeseries-JH.min.json"
+ },
+ {
+   "Abbreviation": "JK",
+   "State": "Jammy and Kashmir",
+   "API": "https://api.covid19india.org/v4/min/timeseries-JK.min.json"
+ },
+ {
+   "Abbreviation": "HP",
+   "State": "Himachal Pradesh",
+   "API": "https://api.covid19india.org/v4/min/timeseries-HP.min.json"
+ },
+ {
+   "Abbreviation": "HR",
+   "State": "Haryana",
+   "API": "https://api.covid19india.org/v4/min/timeseries-HR.min.json"
+ },
+ {
+   "Abbreviation": "GJ",
+   "State": "Gujarat",
+   "API": "https://api.covid19india.org/v4/min/timeseries-GJ.min.json"
+ },
+ {
+   "Abbreviation": "GA",
+   "State": "Goa",
+   "API": "https://api.covid19india.org/v4/min/timeseries-GA.min.json"
+ },
+ {
+   "Abbreviation": "DL",
+   "State": "Delhi",
+   "API": "https://api.covid19india.org/v4/min/timeseries-DL.min.json"
+ },
+ {
+   "Abbreviation": "DN",
+   "State": "Dadra and Nagar Haveli and Daman and Diu",
+   "API": "https://api.covid19india.org/v4/min/timeseries-DN.min.json"
+ },
+ {
+   "Abbreviation": "CT",
+   "State": "Chhattisarh",
+   "API": "https://api.covid19india.org/v4/min/timeseries-CT.min.json"
+ },
+ {
+   "Abbreviation": "CH",
+   "State": "Chandigarh",
+   "API": "https://api.covid19india.org/v4/min/timeseries-CH.min.json"
+ },
+ {
+   "Abbreviation": "BR",
+   "State": "Bihar",
+   "API": "https://api.covid19india.org/v4/min/timeseries-BR.min.json"
+ },
+ {
+   "Abbreviation": "AS",
+   "State": "Assam",
+   "API": "https://api.covid19india.org/v4/min/timeseries-AS.min.json"
+ },
+ {
+   "Abbreviation": "AR",
+   "State": "Arunachal Pradesh",
+   "API": "https://api.covid19india.org/v4/min/timeseries-AR.min.json"
+ },
+ {
+   "Abbreviation": "AP",
+   "State": "Andhra Pradesh",
+   "API": "https://api.covid19india.org/v4/min/timeseries-AP.min.json"
+ },
+ {
+   "Abbreviation": "AN",
+   "State": "Andaman and Nicobar Islands",
+   "API": "https://api.covid19india.org/v4/min/timeseries-AN.min.json"
+ }
+]
```

Comment:
Is this autogenerated or manually done?

Reply:
I autogenerated this from the following:

`<select><option value="{&quot;stateCode&quot;:&quot;TT&quot;,&quot;districtName&quot;:null}">India</option><option value="{&quot;stateCode&quot;:&quot;AN&quot;,&quot;districtName&quot;:null}">Andaman and Nicobar Islands</option><option value="{&quot;stateCode&quot;:&quot;AP&quot;,&quot;districtName&quot;:null}">Andhra Pradesh</option><option value="{&quot;stateCode&quot;:&quot;AR&quot;,&quot;districtName&quot;:null}">Arunachal Pradesh</option><option value="{&quot;stateCode&quot;:&quot;AS&quot;,&quot;districtName&quot;:null}">Assam</option><option value="{&quot;stateCode&quot;:&quot;BR&quot;,&quot;districtName&quot;:null}">Bihar</option><option value="{&quot;stateCode&quot;:&quot;CH&quot;,&quot;districtName&quot;:null}">Chandigarh</option><option value="{&quot;stateCode&quot;:&quot;CT&quot;,&quot;districtName&quot;:null}">Chhattisgarh</option><option value="{&quot;stateCode&quot;:&quot;DN&quot;,&quot;districtName&quot;:null}">Dadra and Nagar Haveli and Daman and Diu</option><option value="{&quot;stateCode&quot;:&quot;DL&quot;,&quot;districtName&quot;:null}">Delhi</option><option value="{&quot;stateCode&quot;:&quot;GA&quot;,&quot;districtName&quot;:null}">Goa</option><option value="{&quot;stateCode&quot;:&quot;GJ&quot;,&quot;districtName&quot;:null}">Gujarat</option><option value="{&quot;stateCode&quot;:&quot;HR&quot;,&quot;districtName&quot;:null}">Haryana</option><option value="{&quot;stateCode&quot;:&quot;HP&quot;,&quot;districtName&quot;:null}">Himachal Pradesh</option><option value="{&quot;stateCode&quot;:&quot;JK&quot;,&quot;districtName&quot;:null}">Jammu and Kashmir</option><option value="{&quot;stateCode&quot;:&quot;JH&quot;,&quot;districtName&quot;:null}">Jharkhand</option><option value="{&quot;stateCode&quot;:&quot;KA&quot;,&quot;districtName&quot;:null}">Karnataka</option><option value="{&quot;stateCode&quot;:&quot;KL&quot;,&quot;districtName&quot;:null}">Kerala</option><option value="{&quot;stateCode&quot;:&quot;LA&quot;,&quot;districtName&quot;:null}">Ladakh</option><option value="{&quot;stateCode&quot;:&quot;MP&quot;,&quot;districtName&quot;:null}">Madhya Pradesh</option><option value="{&quot;stateCode&quot;:&quot;MH&quot;,&quot;districtName&quot;:null}">Maharashtra</option><option value="{&quot;stateCode&quot;:&quot;MN&quot;,&quot;districtName&quot;:null}">Manipur</option><option value="{&quot;stateCode&quot;:&quot;ML&quot;,&quot;districtName&quot;:null}">Meghalaya</option><option value="{&quot;stateCode&quot;:&quot;MZ&quot;,&quot;districtName&quot;:null}">Mizoram</option><option value="{&quot;stateCode&quot;:&quot;NL&quot;,&quot;districtName&quot;:null}">Nagaland</option><option value="{&quot;stateCode&quot;:&quot;OR&quot;,&quot;districtName&quot;:null}">Odisha</option><option value="{&quot;stateCode&quot;:&quot;PY&quot;,&quot;districtName&quot;:null}">Puducherry</option><option value="{&quot;stateCode&quot;:&quot;PB&quot;,&quot;districtName&quot;:null}">Punjab</option><option value="{&quot;stateCode&quot;:&quot;RJ&quot;,&quot;districtName&quot;:null}">Rajasthan</option><option value="{&quot;stateCode&quot;:&quot;SK&quot;,&quot;districtName&quot;:null}">Sikkim</option><option value="{&quot;stateCode&quot;:&quot;TN&quot;,&quot;districtName&quot;:null}">Tamil Nadu</option><option value="{&quot;stateCode&quot;:&quot;TG&quot;,&quot;districtName&quot;:null}">Telangana</option><option value="{&quot;stateCode&quot;:&quot;TR&quot;,&quot;districtName&quot;:null}">Tripura</option><option value="{&quot;stateCode&quot;:&quot;UN&quot;,&quot;districtName&quot;:null}">Unassigned</option><option value="{&quot;stateCode&quot;:&quot;UP&quot;,&quot;districtName&quot;:null}">Uttar Pradesh</option><option value="{&quot;stateCode&quot;:&quot;UT&quot;,&quot;districtName&quot;:null}">Uttarakhand</option><option value="{&quot;stateCode&quot;:&quot;WB&quot;,&quot;districtName&quot;:null}">West Bengal</option></select>`

Reply:
Ah. Do you have the code for this? If it's just a bash script you can even just add to README.

Reply:
done